ci/build-push-multiplatform.sh

```diff
@@ -82,7 +82,7 @@ function build_push() {
     esac
 
     local TAG
-    readonly TAG="${REPO_URL}:${BUILD_TAG}-${BUILD_ARCH}"
+    readonly TAG="${REPO_URL}:${BUILD_TAG}${LATEST_TAG_FIPS_SUFFIX}-${BUILD_ARCH}"
```

Comment:
Does it affect current builds (fips especially)?

Reply:
We can, but what was fips tag before change wand what is it now?

Reply:
Okay, it seems that it does not affect it, but I bugged it and this is why there are failures. I will take care of that.